banking-bench/src/main.rs

```diff
@@ -121,6 +144,34 @@ fn make_accounts_txs(
         .collect()
 }
 
+// In siumulate mint, 99% transactions in a batch are mint transaction (eg., have conflicting account and higher
```

Comment:
Fix the comment - variable % now, not a set 99%.

Reply:
✅